### DISCUSSION TITLE
Interface to kulturpass.de

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretix/pretix#3349] [Q&A]
Hi,
I would like to ask if there will be an interface to kulturpass.de in Pretix, so that tickets can be sold, which can be scanned via Pretix at the entrance.

Thanks a lot!

**Comment by pc-coholic** [comment]
We have been in contact with SAP since the first rumors of the Kulturpass and yes - there will probably be an integration with Kulturpass - at least for one of the proposed purchasing flows and mostly limited to online tickets.

We will post an announcement when the integration is ready for testing on the [pretix.eu Blog](https://pretix.eu/about/de/blog/).

**Comment by chr-chr** [comment]
Hello,

I got this onto my desk today.

Was there any progress on kulturpass?

To my understanding, this can be just selling gift cards. I'm still reading their PowerPoint Files :/

**Reply by raphaelm** [reply]
We are working on an API integration, but it's going to take another few weeks unfortunately.

**Comment by navlisilvan** [comment]
Hi,
will the new Kulturpass-Plugin also be available for self-hosted "Community"-Pretix at some point?

Thanks a lot!

**Reply by raphaelm** [reply]
We have not made a final decision on that yet, for now, it is not available for Community users.
